docs/install/advanced-build-tools-container.md

```diff
@@ -83,7 +83,7 @@ ENTRYPOINT C:\BuildTools\Common7\Tools\VsDevCmd.bat &&
 CMD ["powershell.exe", "-NoLogo", "-ExecutionPolicy", "Bypass"]
 ```
    > [!WARNING]
-   > Visual Studio 2017 version 15.8 or earlier (any product) will not properly install on mcr<span></span>.microsoft\.com\/windows\/servercore:1809 or later. No error is displayed.
+   > Visual Studio 2017 version 15.8 or earlier (any product) will not properly install on mcr\.microsoft\.com\/windows\/servercore:1809 or later. No error is displayed.
```

Comment:
Why is the backslash necessary here but not in docs/install/build-tools-container-issues.md line 33?

Reply:
It isn't, I was just keeping with the style/escaping of the rest of the string